Makefile

```diff
@@ -67,7 +67,11 @@ lint-all: lint-js lint-python lint-html lint-docs lint-docker lint-actions lint-
 lint: lint-js lint-python lint-html lint-links
 lint-js:
 	@yarn eslint --ext js --report-unused-disable-directives "**/*.{js,ts}"
-	@yarn prettier --check "**/*.{js,jsx,ts,tsx,mjs,cjs,mts,cts,md,sql,json,yml,html,css,scss}"
+	@yarn prettier "**/*.{js,jsx,ts,tsx,mjs,cjs,mts,cts,md,sql,json,yml,html,css,scss}" --check
+# This is a separate target due to a cache invalidation bug in prettier (https://github.com/prettier/prettier/issues/17260)
```

Comment:
This isn't true. It also impacts ESLint, and even if Prettier makes the change proposed in that issue, it doesn't solve the issue with cache invalidation in local dev described in https://github.com/PrairieLearn/PrairieLearn/pull/11544/files#r2015065722.

Reply:
Thanks, I will update the comment. 

> # This is a separate target since their caches don't respect updates to prettier or eslint plugins.

Does that copy look ok?

Reply:
LGTM! You could shorten it to "...doesn't respect updates to their plugins" if you want 🤷

Reply:
Resolved in d9ca1c83a193fef790cea46ee9cdedda3a2702a1.